cobbler/modules/managers/import_signatures.py

```diff
@@ -701,7 +704,8 @@ def yum_process_comps_file(self, comps_path, distro):
 
             elif comps_path not in self.found_repos:
                 utils.remove_yum_olddata(comps_path)
-                cmd = "createrepo %s --groupfile %s %s" % (self.settings.createrepo_flags, os.path.join(comps_path, masterdir, comps_file), comps_path)
+                cmd = "createrepo %s --groupfile %s %s" % (
+                self.settings.createrepo_flags, os.path.join(comps_path, masterdir, comps_file), comps_path)
```

Comment:
everywhere else on a multiline you've indented the following line

Reply:
True. Will do. See next force push.